### DISCUSSION TITLE
umi 2.9版本 每次都会自动请求一次本地  但是 部署到线上 请求变为404了 这个怎么解决？

### DISCUSSION THREAD

**Discussion by Hanzo-hss** [umijs/umi#12088] [Q&A]
![WechatIMG161](https://github.com/umijs/umi/assets/85922814/b6e0e669-c0d1-4e64-982d-2c36e7695ca0)

**Comment by fz6m** [comment]
你要找一下这个请求的源头是哪里，一般是你的代码里某处发出的请求，然后去溯源解决它。

umi 2 不维护了，建议升级到 umi 4 。
